### PR TITLE
fix: adding comments

### PR DESCRIPTION
**Pull request by rajesh19998**
this is my initial commit

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
-# Frontend Mentor - Calculator app
+# Frontend Mentor - Calculator app of Rajesh with changes
 
 This is a solution to the [Calculator app challenge on Frontend Mentor](https://www.frontendmentor.io/challenges/calculator-app-9lteq5N29). Frontend Mentor challenges help you improve your coding skills by building realistic projects.
 
-# Table of contents
+# Table of contents this is to follow.
```
Comment (Sindhuja-m1605) [inline README.md]:
```suggestion
# Table of contents.
```
plz remove entire comment

**Review by Sindhuja-m1605 — CHANGES_REQUESTED** [review]
Please address review coments

File: README.md (hunk 1 above)
Comment (rajesh19998) [reply README.md]: @Sindhuja-m1605 i have addressed your review comments. Please review it again.

**Review by Sindhuja-m1605 — APPROVED** [review]
LGTM. Thanks @rajesh19998 for your contribution.

**Pull request merged**